docs/design-documents/docker_management/docker_management.md

```diff
@@ -0,0 +1,135 @@
+# mbed-os-env docker management
+
+
+### Overview and background
+
+docker image mbed-os-env bundles all the necessary tools to provide a minimal environment to build and test mbed-os applications. This docker image shall be used in Continuous Integration pipelines or in other use cases where mbed-os tools and dependencies are required. This document explains versioning strategy of the docker image and github action workflows that creates these docker images.
+
+
+### Types of docker images
+
+**Production docker image**
+These are docker images compatible with a released version of mbed-os. For example when `mbed-os-6.14.0` is released, a docker image with tag `mbed-os-6.14-latest` is available.
```

Comment:
Why not have `mbed-os-6.14:latest` ?

Reply:
the image will be `mbed-os-env:mbed-os-6.14-latest` 
`mbed-os-6.14-latest` being the tag here.